content/tutorials/manage-account-settings/use-tickets.md

```diff
@@ -0,0 +1,372 @@
+---
+html: use-tickets.html
+funnel: Build
+doc_type: Tutorials
+category: Manage Account Settings
+blurb: Use Tickets to send a transaction outside of normal Sequence order.
+filters:
+    - interactive_steps
+---
+# Use Tickets
+
+_(Requires the [TicketBatch amendment][] :not_enabled:)_
+
+[Tickets](tickets.html) provide a way to send transactions out of the normal order. This tutorial walks through the steps of creating a Ticket, then using it to send another transaction.
+
+## Prerequisites
+
+<!-- Interactive example use ripple-lib and its prerequisites -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.11/lodash.js"></script>
+<script type="application/javascript" src="{{target.ripple_lib_url}}"></script>
+<!-- Helper for interactive tutorial breadcrumbs -->
+<script type="application/javascript" src="assets/js/interactive-tutorial.js"></script>
+<!-- Source for this specific tutorial's interactive bits: -->
+<script type="application/javascript" src="assets/js/tutorials/use-tickets.js"></script>
+
+This page provides JavaScript examples that use the ripple-lib (RippleAPI) library. The [RippleAPI Beginners Guide](get-started-with-rippleapi-for-javascript.html) describes how to get started using RippleAPI to access XRP Ledger data from JavaScript.
+
+Since JavaScript works in the web browser, you can read along and use the interactive steps without any setup.
+
+Tickets must be enabled. At this time, the [TicketBatch amendment][] :not_enabled: is only available on Devnet.
+
+
+
+## Steps
+{% set n = cycler(* range(1,99)) %}
+
+### {{n.next() }}. Get Credentials
+
+To interact with the XRP Ledger, you need a address and secret key, and some XRP. You can get all of these on the [Devnet](parallel-networks.html) using the following interface: <!--TODO: change to Testnet eventually -->
```

Comment:
If you have some XRP then you probably have an address and secret key. There are some exceptions, but they are rare, so I think we can simplify to:
```suggestion
To transact on the XRP Ledger, you need some XRP. For development purposes, you can get some free test XRP on the [Devnet](parallel-networks.html) using the following interface: <!--TODO: change to Testnet eventually -->
```

Reply:
> If you have some XRP then you probably have an address and secret key.

I hear a lot of people say they have XRP, but what they mean is they hold have some at a custodial exchange, which may even give them a deposit address (even more likely if X-addresses gain more adoption) but not a secret.

Reply:
Good point. Here's a smaller suggested change:
```suggestion
To transact on the XRP Ledger, you need an address and secret key, and some XRP. For development purposes, you can get these on the [Devnet](parallel-networks.html) using the following interface: <!--TODO: change to Testnet eventually -->
```